zadanie 7.2

Frodo1996 opened the pull request.

Frodo1996 committed:
zadanie 7.2

marcinkunert reviewed (changes requested):
Film z code review: https://youtu.be/fdSAPgb4Ni0

Frodo1996 committed:
z tego co zrozumiałem zmienna "itIs" okazała się zbędną.
Przetestowałem działanie programu usuwając ją i wszystko działa jak powinno, także liczę, że rozwiązanie zadania jest odpowiednie, pozdrawiam
zadanie 7.2

marcinkunert commented:
Nie jest zbędna. Po prostu nigdy nie przestawiałeś jej z powrotem na `false`.
